vcsutils/webhookparser/webhookparser.go

```diff
@@ -0,0 +1,40 @@
+package webhookparser
+
+import (
+	"net/http"
+
+	"github.com/jfrog/froggit-go/vcsutils"
+)
+
+const EventHeaderKey = "X-Event-Key"
+
+type WebhookInfo struct {
+	Repository       string                `json:"repository,omitempty"`        // The target repository on pull requests and push
+	Branch           string                `json:"branch,omitempty"`            // The target branch on pull requests and push
+	SourceRepository string                `json:"source_repository,omitempty"` // The source repository on pull requests
+	SourceBranch     string                `json:"source_branch,omitempty"`     // The source branch on pull requests
+	Timestamp        int64                 `json:"timestamp,omitempty"`         // Seconds from epoch
+	Event            vcsutils.WebhookEvent `json:"event,omitempty"`             // The event type
+}
+
+type WebhookParser interface {
+	validatePayload(token []byte) ([]byte, error)
```

Comment:
Let's add a description for this function for this fucntion, as well as the returned value. It would be good to include this description in all the implemenattions as well.
The same goes for the below ```parseIncomingWebhook``` method.